src/shims/time.rs

```diff
@@ -40,9 +38,10 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
             [this.eval_libc_i32("CLOCK_MONOTONIC")?, this.eval_libc_i32("CLOCK_MONOTONIC_COARSE")?];
 
         let duration = if absolute_clocks.contains(&clk_id) {
+            this.check_no_isolation("`clock_gettime` with real time clocks")?;
```

Comment:
```suggestion
            this.check_no_isolation("`clock_gettime` with REALTIME clocks")?;
```
The use of lowercase "real" here is a bit ambiguious because there is the real-ness of the argument to `clock_gettime` and the real-ness of Miri's clock.